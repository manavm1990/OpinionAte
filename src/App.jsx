import React from "react";

function App() {
  return (
    <h1 className="capitalize mt-3 text-3xl text-center text-red-500">
      {" "}
      Hello world!{" "}
    </h1>
  );
}

export default App;
