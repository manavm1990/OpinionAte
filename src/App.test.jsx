import React from "react";
import renderer from "react-test-renderer";
import App from "./App";

describe("My Test Suite", () => {
  it("My Test Case", () => {
    expect(true).toBe(true);
  });
});

describe("App", () => {
  test("snapshot renders", () => {
    const component = renderer.create(<App />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
