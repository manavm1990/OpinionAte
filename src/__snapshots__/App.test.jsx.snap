// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot renders 1`] = `
<h1
  className="capitalize mt-3 text-3xl text-center text-red-500"
>
   
  Hello world!
   
</h1>
`;
